exporter/kineticaexporter/factory.go

```diff
@@ -0,0 +1,120 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kineticaexporter
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/exporter"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+// NewFactory creates a factory for Kinetica exporter.
+//
+//	@return exporter.Factory
+func NewFactory() exporter.Factory {
+	return exporter.NewFactory(
+		typeStr,
+		CreateDefaultConfig,
+		exporter.WithLogs(createLogsExporter, component.StabilityLevelDevelopment),
+		exporter.WithTraces(createTracesExporter, component.StabilityLevelDevelopment),
+		exporter.WithMetrics(createMetricsExporter, component.StabilityLevelDevelopment),
+	)
+}
+
+// CreateDefaultConfig - function to create a default configuration
+//
+//	@return component.Config
+func CreateDefaultConfig() component.Config {
+	return &Config{
+		Host:               "http://localhost:9191/",
+		Schema:             "",
+		Username:           "admin",
+		Password:           "password",
```

Comment:
Are these meaningful defaults? Could the exporter connect to a database running at `http://localhost:9191/` with these credentials?

Reply:
Replaced them with what we consider as meaningful defaults which are empty strings for both values.